modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,4415 @@
+
+| `vertexName`
+| yes
+| the vertex types to be delete, if there exist more than one, use , separate it. value all means dorp all the vertices
+|===
+
+'''  
+
+=== drop local vertices
+
+`DELETE /gsql/v1/schema/vertices`
+
+This endpoint is used to drop local vertices.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X DELETE -H "content-type: text/plain" "http://localhost:14240/gsql/v1/schema/vertices?vertexName=user2,user3&graph=recommend"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Vertices [user2, user3] deleted successfully."}
+----
+--
+====
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `vertexName`
+| yes
+| the vertex types to be delete, if there exist more than one, use , separate it. value all means dropping all the vertices
+
+| `graph`
+| yes
+| the vertices in which graph to be deleted
+|===
+
+'''  
+
+=== drop a local vertex
+
+`DELETE /gsql/v1/schema/vertices/{vertexName}`
+
+This endpoint is used to drop a local vertex.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X DELETE -H "content-type: text/plain" "http://localhost:14240/gsql/v1/schema/vertices/user2?graph=recommend"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Vertices [user2] deleted successfully."}
+----
+--
+====
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the vertex in which graph to be deleted
+|===
+
+
+'''  
+=== drop a global vertex
+
+`DELETE /gsql/v1/schema/vertices/{vertexName}`
+
+This endpoint is used to drop a global vertex.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X DELETE -H "content-type: text/plain" "http://localhost:14240/gsql/v1/schema/vertices/user2"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Vertices [user2] deleted successfully."}
+----
+--
+====
+==== Parameters:
+
+None
+
+'''  
+=== update vertex attribute (local graph)
+
+`PUT /gsql/v1/schema/vertices/{vertexName}`
+
+This endpoint is used to update a vertex attributes.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X PUT -H "content-type: application/json" "http://localhost:14240/gsql/v1/schema/vertices/user?graph=recommend" -d '{"dropAttributes":["area"],"addAttributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"attr1"}]}'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully update vertex: user"}
+----
+--
+====
+==== Parameters:
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the vertex in which graph to update
+|===
+
+'''  
+=== update a global vertex attribute
+
+`PUT /gsql/v1/schema/vertices/{vertexName}`

```

Comment:
Ditto and same for the following part, combine them if their API are the same. This page is huge, we probably do not want to make it even more complicated.